src/modules/GroupChannel/context/utils.ts

```diff
@@ -35,3 +35,8 @@ export const isDisabledBecauseMuted = (groupChannel?: GroupChannel | null) => {
   if (!groupChannel) return false;
   return groupChannel.myMutedState === 'muted';
 };
+
+export const isDisabledBecauseSuggestedReplies = (channel: GroupChannel | null | undefined, enableSuggestedReplies: boolean) => {
+  if (!channel || !channel.lastMessage) return false;
+  return enableSuggestedReplies && !!channel.lastMessage.extendedMessagePayload?.disable_chat_input;
```

Comment:
```suggestion
  return enableSuggestedReplies && !!channel?.lastMessage?.extendedMessagePayload?.disable_chat_input;
```
Actually we can get a same result even without the first line with the optional chainings.

Reply:
-> https://github.com/sendbird/sendbird-uikit-react/pull/1137/commits/4fc8b62cc102f249795669021d115a01b17fb88c